specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/ServerDevOpsAuditGet.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "devAuditTestRG",
+    "serverName": "devOpsAuditTestSvr",
+    "DevOpsAuditPolicyName": "default",
+    "api-version": "2020-02-02-preview",
+    "devOpsAuditPolicyName": "default"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/devAuditTestRG/providers/Microsoft.Sql/servers/devOpsAuditTestSvr/devOpsAuditingSettings/default",
+        "name": "default",
+        "type": "Microsoft.Sql/servers/devOpsAuditingSettings",
+        "properties": {
+          "state": "Disabled",
+          "storageEndpoint": "",
+          "storageAccountSubscriptionId": "00000000-0000-0000-0000-000000000000",
```

Comment:
If state is disabled, shouldn't the sub id be null rather than Guid.Empty?

Reply:
well, not necessarily.
when customer set the policy to disabled, we may keep some param as part of the policy.
such thing is useful for UI and PS ( there is no need to configure all audit properties all over again).

Reply:
Just to be a bit more clear, policy can be disabled and still contains a storageAccountSubscriptionId (a valid one).

Reply:
I see ok